src/mesh/meshblock.cpp

```diff
@@ -218,8 +218,17 @@ void MeshBlock::InitializeIndexShapesImpl(const int nx1, const int nx2, const in
 
   if (init_coarse) {
     if (multilevel) {
+      // Prevent the coarse bounds from going to zero
+      int cnx1 = nx1 / 2;
+      int cnx2 = nx2 / 2;
+      int cnx3 = nx3 / 2;
+      if (pmy_mesh != nullptr) {
```

Comment:
Under which conditions can the mesh be a `nullptr`?

Reply:
In the unit tests. I didn't add it initially and many of them segfaulted.